components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/launcher/launcher.js

```diff
@@ -65,25 +65,65 @@ define(['log', 'jquery', 'backbone', 'lodash', 'context_menu', 'launch_manager',
 
             runApplication: function (workspace, async) {
                 var activeTab = this.application.tabController.getActiveTab();
-                // only saved files can be run as application
-                if (this.isReadyToRun(activeTab)) {
-                    var siddhiAppName = "";
-                    if(activeTab.getTitle().lastIndexOf(".siddhi") != -1){
-                        siddhiAppName = activeTab.getTitle().substring(0, activeTab.getTitle().lastIndexOf(".siddhi"));
-                    } else{
-                        siddhiAppName = activeTab.getTitle();
+                var outputController = this.application.outputController;
+                var commandManager = this.application.commandManager;
+                var editorText = activeTab.getFile().getContent();
+                console.log(editorText);
```

Comment:
Remove the console logs

Reply:
Fixed in https://github.com/wso2/carbon-analytics/pull/1870/commits/83fe82d8ade54e03ed8f13e13ed86528d003fbc5